src/lib/ledgers/five-bells-ledger.js

```diff
@@ -146,18 +134,24 @@ FiveBellsLedger.prototype._autofund = function * () {
       connector: config.getIn(['server', 'base_uri']),
       password: this.credentials.password,
       fingerprint: this.credentials.fingerprint
-    },
-    cert: admin.cert,
-    ca: admin.ca,
-    key: admin.key
-  }, 'could not create account at ledger ' + this.id)
+    }
+  }, 'could not create account at ledger ' + this.id, admin)
 }
 
-function * request_retry (opts, error_msg) {
+function * request_retry (opts, error_msg, credentials) {
   let delay = backoffMin
   while (true) {
     try {
-      let res = yield request(opts)
+      let res = yield request(lodash.defaults(opts, {
+        auth: credentials && {
```

Comment:
This was like this before I made this change:

For admin it's user/pass
https://github.com/interledger/five-bells-connector/blob/master/src/lib/config.js#L58

For ledger credentials it's username/password
https://github.com/interledger/five-bells-connector/#trading

How is it possible to send basic-auth of undefined:undefined...?